src/components/ListItem/example.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ListItem } from '.';
+
+
+export default function ListItemExample({ clickHandler }) {
+  const onClick = () => clickHandler('onClick', 'clicked it');
```

Comment:
The idea behind the `clickHandler` was to use it directly on the example components, so we can see what they return to the callback, if anything.